src/NuGet.Clients/NuGet.SolutionRestoreManager/VsSolutionRestoreService.cs

```diff
@@ -317,9 +318,13 @@ private static IEnumerable<string> GetRestoreSources(IVsTargetFrameworks tfms)
         {
             var sources = HandleClear(MSBuildStringUtility.Split(GetNonEvaluatedPropertyOrNull(tfms, RestoreSources, e => e)));
 
-            var additional = MSBuildStringUtility.Split(GetNonEvaluatedPropertyOrNull(tfms, RestoreAdditionalProjectSources, e => e));
+            // Read RestoreAdditionalProjectSources from the inner build, these may be different between frameworks.
+            // Exclude is not allowed for sources
+            var additional = MSBuildRestoreUtility.AggregateSources(
```

Comment:
>AggregateSources [](start = 51, length = 16)

AggregateEntries or something maybe?
Because it's called for both sources/fallback folders